src/renderer/api/endpoints/deployment.api.ts

```diff
@@ -23,6 +25,25 @@ export class DeploymentApi extends KubeApi<Deployment> {
       }
     })
   }
+
+  restart(params: { namespace: string; name: string }) {
+    return this.request.patch(this.getUrl(params), {
+      data: {
+        spec: {
+          template: {
+            metadata: {
+              annotations: {"kubectl.kubrnetes.io/restartedAt" : moment.utc().format()}
```

Comment:
Typo in annotation? `kubectl.kubrnetes.io/restartedAt` -> `kubectl.kubernetes.io/restartedAt`

Reply:
oh, year, fixed